drivers/gles3/shaders/scene.glsl

```diff
@@ -1666,20 +1666,32 @@ void reflection_process(samplerCube reflection_map,
 	vec3 ref_normal = normalize(reflect(vertex, normal));
 	ref_normal = (local_matrix * vec4(ref_normal, 0.0)).xyz;
 
+	float mip = sqrt(roughness) * MAX_ROUGHNESS_LOD;
+	float mip_min = pow(roughness, 2.0) * MAX_ROUGHNESS_LOD; // Ensures fully rough materials don't have reflection contact hardening.
```

Comment:
```suggestion
	float mip_min = (roughness * roughness) * MAX_ROUGHNESS_LOD; // Ensures fully rough materials don't have reflection contact hardening.
```
I would feel better if you avoided the pow function in this case. Sure, some compilers might recognize that this has an integer exponent of two, but I would rather not rely on that.